www/data.php

```diff
@@ -374,6 +374,11 @@ classiques :
     "http://www.april.org",
     "Pionnière du logiciel libre en France, l'April est depuis 1996 un acteur majeur de la démocratisation et de la diffusion du Logiciel Libre et des standards ouverts auprès du grand public, des professionnels et des institutions dans l'espace francophone. Elle veille aussi à sensibiliser l'opinion sur les dangers d'une appropriation exclusive de l'information et du savoir par des intérêts privés."
   ),
+  array("Fédération Open Space Maker",
+    "img/fosm.png",
```

Comment:
Il y a un petit soucis avec le logo

```suggestion
    "img/Federation-small.png",
```